app/tool/tasks.cxx

```diff
@@ -550,11 +567,14 @@ static void parameter_timescan(Target* tgt) {
     // Add implementation for other pages as well
     for (std::size_t i_param{0}; i_param < param_names.size(); i_param++) {
       test_param_builder.add(
-      (param_names[i_param].first == "REFERENCEVOLTAGE_0" ? refvol_page : 
-       param_names[i_param].first == "REFERENCEVOLTAGE_1" ? refvol_page :
-       param_names[i_param].first),
+      "REFERENCEVOLTAGE_0",
```

Comment:
This change breaks the ability for a parameter in the input parameter points file to be from a page that isn't `REFERENCEVOLTAGE` (`0` or `1`). Why did you make this change? What was the motivation?

If you need to only be changing the RV page parameters, then maybe you should make a new command instead of trying to make this (already pretty complicated) command work for your purposes?